pwa/app/components/tba/nav.tsx

```diff
@@ -8,6 +8,7 @@ import BiPeopleFill from '~icons/bi/people-fill';
 import BiStarFill from '~icons/bi/star-fill';
 import BiThreeDotsVertical from '~icons/bi/three-dots-vertical';
 import IonCalendar from '~icons/ion/calendar';
+import SearchIcon from '~icons/lucide/search';
```

Comment:
Sorry to be super nitpicky here; does [`icons/bi/search`](https://icon-sets.iconify.design/bi/search/) fit the UI better? I only ask because we use bootstrap icons in a lot of places (but not all, sometimes there are better alternatives)

Not a blocking comment, just curious what you think

Reply:
It's all good :) Yes, I tried the `bi` icons but they seem to me to be very old, thin and not compatible for the new interface. That's why I'm in favor of switching to another icons pack - just because we used Bootstrap in the original web doesn't mean the new design has to be completely same and not modern.
 
You might see the difference in #6453.